### PR TITLE
add uri-template 1.1.0

### PR DESCRIPTION
**Pull request by bollwyvl**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

Notes:
- blocks https://github.com/conda-forge/jsonschema-feedstock/issues/27

### CODE REVIEW

**Comment by bollwyvl** [comment]
@conda-forge/staged-recipes @conda-forge/help-python ready for review

File: recipes/uri-template/fix-version.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+import os
+
+SRC_DIR = Path(os.environ["SRC_DIR"])
+SETUP_PY = SRC_DIR / "setup.py"
+UTF8 = dict(encoding="utf-8")
+PKG_VERSION = os.environ["PKG_VERSION"]
+
+SETUP_PY.write_text(
+    SETUP_PY.read_text(**UTF8).replace(
+        '''version='0.0.0',''',
```
Comment (ocefpaf) [inline recipes/uri-template/fix-version.py:11]: Fun!

File: recipes/uri-template/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,53 @@
+{% set version = "1.1.0" %}
+
+package:
+  name: uri-template
+  version: {{ version }}
+
+source:
+  url: https://gitlab.linss.com/open-source/uri_template/-/archive/v1.1.0/uri_template-v{{ version }}.tar.gz
```
Comment (ocefpaf) [inline recipes/uri-template/meta.yaml:8]:
```suggestion
  url: https://gitlab.linss.com/open-source/uri_template/-/archive/v{ version }/uri_template-v{{ version }}.tar.gz
```

File: recipes/uri-template/fix-version.py (hunk 1 above)
Comment (bollwyvl) [reply recipes/uri-template/fix-version.py:11]: :crying_cat_face:

File: recipes/uri-template/meta.yaml (hunk 2 above)
Comment (bollwyvl) [reply recipes/uri-template/meta.yaml]: thanks!

**Comment by bollwyvl** [comment]
@conda-forge/staged-recipes @conda-forge/help-python properly-parametrized URL, and back up to ready for review. thanks again!

**Comment by bollwyvl** [comment]
Thanks!

**Pull request merged**